odoo/addons/base/models/res_users.py

```diff
@@ -691,267 +822,6 @@ def write(self, values):
-
-    @api.model
-    def create(self, values):
-        user = super(GroupsView, self).create(values)
-        self._update_user_groups_view()
-        # actions.get_bindings() depends on action records
-        self.env['ir.actions.actions'].clear_caches()
-        return user
-
-    @api.multi
-    def write(self, values):
-        res = super(GroupsView, self).write(values)
-        self._update_user_groups_view()
-        # actions.get_bindings() depends on action records
-        self.env['ir.actions.actions'].clear_caches()
-        return res
-
-    @api.multi
-    def unlink(self):
-        res = super(GroupsView, self).unlink()
-        self._update_user_groups_view()
-        # actions.get_bindings() depends on action records
-        self.env['ir.actions.actions'].clear_caches()
-        return res
-
-    @api.model
-    def _update_user_groups_view(self):
-        """ Modify the view with xmlid ``base.user_groups_view``, which inherits
-            the user form view, and introduces the reified group fields.
-        """
-        if self._context.get('install_mode'):
-            # use installation/admin language for translatable names in the view
-            user_context = self.env['res.users'].context_get()
-            self = self.with_context(**user_context)
-
-        # We have to try-catch this, because at first init the view does not
-        # exist but we are already creating some basic groups.
-        view = self.env.ref('base.user_groups_view', raise_if_not_found=False)
-        if view and view.exists() and view._name == 'ir.ui.view':
-            group_no_one = view.env.ref('base.group_no_one')
-            xml1, xml2 = [], []
-            xml1.append(E.separator(string=_('Application Accesses'), colspan="2"))
-            for app, kind, gs in self.get_groups_by_application():
-                # hide groups in categories 'Hidden' and 'Extra' (except for group_no_one)
-                attrs = {}
-                if app.xml_id in ('base.module_category_hidden', 'base.module_category_extra', 'base.module_category_usability'):
-                    attrs['groups'] = 'base.group_no_one'
-
-                if kind == 'selection':
-                    # application name with a selection field
-                    field_name = name_selection_groups(gs.ids)
-                    xml1.append(E.field(name=field_name, **attrs))
-                    xml1.append(E.newline())
-                else:
-                    # application separator with boolean fields
-                    app_name = app.name or _('Other')
-                    xml2.append(E.separator(string=app_name, colspan="4", **attrs))
-                    for g in gs:
-                        field_name = name_boolean_group(g.id)
-                        if g == group_no_one:
-                            # make the group_no_one invisible in the form view
-                            xml2.append(E.field(name=field_name, invisible="1", **attrs))
```

Comment:
The group_no_one should be invisible, not hidden in debug mode.

Reply:
By using the option on the settings: https://drive.google.com/file/d/1au7jHfi6vFZr4rL9tnneX9Mk_f3sE246/view?usp=drivesdk

This group is a technical one. Used to show debugging fields. We don't want a user to be always in this group.